### PR TITLE
My very first PR.

### PR DESCRIPTION
**Pull request by FirinMaLazors**
Adds a description to the APC!

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Power/saltern_power.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 - type: entity
   id: SalternApc
+  description: A control terminal for the area electrical systems.
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Constructible/Power/saltern_power.yml:37]: Maybe `area's` here instead?
```suggestion
  description: A control terminal for the area's electrical systems.
```

Comment (FirinMaLazors) [reply Resources/Prototypes/Entities/Constructible/Power/saltern_power.yml:37]: Sure!  I literally just copied it from the Aurorastation code, so *shrug*.  I can see a grammatical case for the former, but whatever works : D

**Comment by SweptWasTaken** [comment]
Can someone please fucking merge this shit instead of attaching labels to it for fucks sake this it a no-brainer AGRSGHHGRHGH

**Comment by PaulRitter** [comment]
lmao swept is right, this is literally a one-line change

**Comment by moonheart08** [comment]
@SweptWasTaken nothing escapes the bikeshed.

**Comment by SweptWasTaken** [comment]
The battle is won

**Pull request merged**